src/libcollections/fmt.rs

```diff
@@ -115,13 +115,8 @@
 //!
 //! If this syntax is used, then the number of characters to print precedes the
 //! actual object being formatted, and the number of characters must have the
-//! type `usize`. Although a `usize` can be printed with `{}`, it is invalid to
```

Comment:
Indeed, I opted to remove this because it confused both nodakai and me. I'll try to expand it.